modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/processor/PartitionReconciliationProcessor.java

```diff
@@ -68,19 +70,37 @@ public class PartitionReconciliationProcessor extends AbstractPipelineProcessor
     /** Topology change message. */
     public static final String TOPOLOGY_CHANGE_MSG = "Topology has changed. Partition reconciliation task was stopped.";
 
-    /** Work progress message. */
-    public static final String WORK_PROGRESS_MSG = "Partition reconciliation task [sesId=%s, total=%s, remaining=%s]";
-
     /** Start execution message. */
-    public static final String START_EXECUTION_MSG = "Partition reconciliation has started [repair=%s, repairAlg=%s, " +
+    private static final String START_EXECUTION_MSG = "Partition reconciliation has started [sesionId=%s, repair=%s, repairAlgorithm=%s, " +
         "fastCheck=%s, batchSize=%s, recheckAttempts=%s, parallelismLevel=%s, caches=%s]";
 
+    /** Stop execution message. */
+    private static final String STOP_EXECUTION_MSG = "Partition reconciliation has finished locally [sessionId=%s, conflicts=%s, " +
+        "repaired=%s, totalTime=%s(sec)]";
+
+    /** Session progress message. */
+    private static final String SESSION_PROGRESS_MSG = "Partition reconciliation status [" +
+        "sesId=%s, " +              // session identifier
+        "caches=%s, " +             // total number of caches to be scanned, this number may be less than number of caches requested
+                                    // by the tool due to the fact that some caches might be skipped because of the expiration policy or
+                                    // the node does not own primary partitions of the cache
+        "parts= %s, " +             // total number of primary partitions to be scanned
```

Comment:
Is it an immutable value? Meaning that we will not adjust it while processing.

Reply:
yep, you are correct.